lib/archethic/metrics/poller.ex

```diff
@@ -0,0 +1,82 @@
+defmodule ArchEthic.Metrics.Poller do
+  @moduledoc """
+  Provides Telemetry of the network and Maneges
+  """
+  require Logger
+  use GenServer
+  # map because you cant pop from linked list
+  @default_state %{
+    pid_refs: %{},
+    data: ArchEthic.Metrics.Helpers.get_client_metric_default_value()
+  }
+
+  def start_link(_state) do
+    GenServer.start_link(__MODULE__, @default_state, name: __MODULE__)
+  end
+
+  def init(initial_state) do
+    periodic_metric_aggregation()
+    periodic_push_updates()
+    {:ok, initial_state}
+  end
+
+  def periodic_metric_aggregation() do
+    Process.send_after(self(), {:periodic_calculation_of_points}, 5_000)
+  end
+
+  def periodic_push_updates() do
+    Process.send_after(self(), :push_updates, 251)
+  end
+
+  def send_updates(%{data: data, pid_refs: pid_refs}) do
+    Enum.each(pid_refs, fn {pid_k, _pid_v} ->
+      Task.start(fn ->
```

Comment:
Pub sub was avoided due to one by one dispatch. 
If .1 million requests are there so the updates can be pushed quickly. , parallelly.
I am assuming that task is used for parallel workload with no link to any process.

Reply:
This `send_updates` function is only called when the poller finished its job, so it will be called once every 5s.
But if you want to send in concurrency, we can `Task.async_stream` which is more suitable for that.

Reply:
In this situation its required that :
-deploy  message with no expected return value /message  and with no process link
Will Task. async _ stream _ start  _ nolink will be better???or does it returns something.???

Reply:
Hi , 
I tried with Task.async_stream  the live process  does not update.
what should I use ?

Reply:
🤦 my bad I forgot to use stream.run